bugbug/bugzilla.py

```diff
@@ -93,20 +100,16 @@
 
 INCLUDE_FIELDS = ["_default", "filed_via"]
 
-PRODUCTS_TO_EXCLUDE = [
-    "Calendar",
-    "Chat Core",
-    "Invalid Bugs",
-    "MailNews Core",
-    "Thunderbird",
-]
-
 
-def get_bugs(include_invalid: bool | None = False) -> Iterator[BugDict]:
+def get_bugs(
+    include_invalid: bool | None = False, include_mail_products: bool = False
```

Comment:
Instead of a specific "include_mail_products" which is either True or False, it would be more generic to have something like "include_additional_products" as a list, and the caller can specify the list.